keyboards/kbdfans/kbd75/keymaps/aaronireland/keymap.c

```diff
@@ -0,0 +1,199 @@
+    KC_LSFT,          KC_SCLN, KC_Q   , KC_J   , KC_K   , KC_X   , KC_B   , KC_M   , KC_W   , KC_V   , KC_Z   , KC_RSFT,          KC_UP  , KC_END ,
+    KC_LGUI, KC_LALT, KC_LCTL,                   KC_SPC ,                                     KC_RGUI, KC_LCTL, MO(_FL), KC_LEFT, KC_DOWN, KC_RGHT
+  ),
+
+  /*   ******************** 3: FN Momentary Layer ********************
+   *  F13   - Launchpad
+   *  SNU   - CTRL + B = Neovim Screen Up
+   *  SND   - CTRL + D = Neovim Screen Down
+   *  TOG   - Toggle RGB Underglow
+   *  MOD   - RGB Underglow Mode
+   *  NXT   - FF/Skip
+   *  PLY   - Play/Stop
+   *  VL    - Volume Down/Up
+   *  SN    - Screen Brightness Up/Down
+   *  QRT   - QWERTY Layout
+   *  CLK   - COLEMAK Layout
+   *  DVK   - DVORAK Layout
+   *  RESET - Put PCB into Bootstrap mode
+   * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+   * │   │   │   │   │   │   │   │   │   │   │   │   │   │PSN│   │   │
+   * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┴───┼───┤
+   * │   │TOG│MOD│HU+│HU-│SA+│SA-│VA+│VA-│   │   │   │   │ RESET │F13│
+   * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┼───┤
+   * │ TAB │QRT│   │   │   │   │   │   │   │   │   │   │   │     │SNU│
+   * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+   * │ CAPS │   │   │DVK│   │   │   │   │   │   │   │   │        │SND│
+   * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+   * │        │   │   │CLK│DEC│TOG│INC│STP│   │   │   │      │SN+│VL0│
+   * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴──┬┴──┬┴──┬───┼───┼───┤
+   * │ OS │ ALT│CTRL│          SPACE         │NXT│PLY│XXX│VL-│SN-│VL+│
+   * └────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┴───┘
+   *   ************************************************************/
+
+  [_FL] = LAYOUT_ansi_1u(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_CAPP, _______, _______,
+    _______, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, _______, _______, _______, RESET  ,          KC_F13 ,
+    KC_TAB , QWERTY , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          KC_SNU ,
+    KC_CAPS, _______, _______, DVORAK , _______, _______, _______, _______, _______, _______, _______, _______,          _______,          KC_SND ,
+    _______,          _______, _______, COLEMAK, BL_DEC , BL_TOGG, BL_INC , BL_STEP, _______, _______, _______, _______,          KC_PAUS, KC_MUTE,
+    KC_LGUI, KC_LALT, KC_LCTL,                   KC_SPC ,                                     KC_MNXT, KC_MPLY, _______, KC_VOLD, KC_SLCK, KC_VOLU
+  ),
+};
+
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Reply:
Done. Thanks for the review!